app/utils/av-scan/get-av-token.js

```diff
@@ -0,0 +1,32 @@
+const Wreck = require('@hapi/wreck')
+const avConfig = require('../../config/index')
+
+const getTokenRequestBody = new URLSearchParams({
+  grant_type: avConfig.grantType,
+  client_id: avConfig.clientId,
+  client_secret: avConfig.clientSecret,
+  scope: avConfig.scope
+}).toString()
+
+const getTokenHeaders = {
+  'Content-Type': 'application/x-www-form-urlencoded'
+}
+
+async function getAVToken () {
```

Comment:
We typically use arrow functions over declared async functions, so for example this might instead be written as:
```
const getAVToken = () => {
  try {
      const { res, payload } = await Wreck.post(avConfig.tokenUrl, {
        payload: getTokenRequestBody,
        headers: getTokenHeaders
      })
      if (res.statusCode !== 200) {
        throw new Error(`Token Request Failed: ${res.statusCode} ${res.statusMessage}`)
      }
      const token = JSON.parse(payload.toString('utf8'))
      return `${token.token_type} ${token.access_token}`
    } catch (error) {
      console.error('ERROR in getting AV token:', error)
      throw error
    }
}
```
Everything remains the same, just changing line 15 essentially. Unless of course there's a specific reason you chose to use a declared async function?